apps/tup-ui/src/main.global.css

```diff
@@ -45,6 +45,10 @@ h2 {
   font-size: var(--global-font-size--large);
   font-weight: var(--bold);
 }
+h3 {
+  font-size: var(--global-font-size--medium);
+  font-weight: var(--bold);
+}
```

Comment:
I am adding these title styles as needed.
<sup>I plan to move them to [Core-Styles > Elements > Headings > Portal](https://tacc.utexas.edu/static/ui/components/detail/headings--portal.html) (currently un-styled) when Core Portal as ready for them.<sup>